src/pages/ChooseArea.svelte

```diff
@@ -142,40 +151,52 @@
   }
 </script>
 
-<div class="left">
-  <h1>Welcome to ATIP v2</h1>
-  <button type="button" on:click={() => (showAbout = !showAbout)}>About</button>
-
-  <p>Select Transport Authority or Local Authority District:</p>
-  <div>
-    <input
-      data-testid="transport-authority"
-      list="authorities-list"
-      bind:value={inputValue}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half left">
+    <h1 class="govuk-heading-l">Welcome to ATIP v2</h1>
+    <SecondaryButton on:click={() => (showAbout = !showAbout)}
+      >About</SecondaryButton
+    >
+
+    <FormElement
+      label="Select Transport Authority or Local Authority District"
+      id="inputValue"
+    >
+      <input
+        class="govuk-input govuk-input--width-20"
+        id="inputValue"
+        data-testid="transport-authority"
+        list="authorities-list"
+        bind:value={inputValue}
+      />
+      <datalist id="authorities-list" bind:this={dataList} />
+    </FormElement>
+    <DefaultButton on:click={start} disabled={!validEntry}>Start</DefaultButton>
+
+    <hr />
+
+    <Radio
+      legend="Or pick from the map"
+      id="showBoundaries"
+      choices={[
```

Comment:
This used to be a select/dropdown, but https://design-system.service.gov.uk/components/select/ recommends against, and I agree here